doc/replication-policies.md

```diff
@@ -0,0 +1,197 @@
+# Replication Policies
+
+## iOS replication policies
```

Comment:
Since most of the APIs for polices can be used on Mac too, I think we need to make it clear it's a cross platform feature, not sure how we'd go about it though